### PR TITLE
Add `send` event to each enhanced commerce event

### PR DESCRIPTION
**Pull request by brunojdo**
https://analytics--storecomponents.myvtex.com/long-sleeve-shirt/p

### CODE REVIEW

**Review by klzns — APPROVED** [review]
Looks beautiful and refreshing ✨

File: react/components/enhancedCommerce.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+export const productDetail = (product: any) => {
+  ga('ec:addProduct', {
+    brand: product.brand,
+    category: product.categoryId,
+    id: product.productId,
+    name: product.productName,
+    variant: product.variant,
+  })
+  ga('ec:setAction', 'detail')
+  ga('send', 'event', 'Ecommerce', 'Detail', { nonInteraction: 1 })
```
Comment (italo-batista) [inline react/components/enhancedCommerce.ts]: Maybe it would be better to have the event _category_ Ecommerce -> Product and the _action_ Detail -> View.

**Review by italo-batista — APPROVED** [review]
Approved but with just one comment

**Review by lucasecdb — COMMENTED** [review]
nitpick: why the event functions are in a `components` folder? :thinking:

I think it would make more sense for a `events` folder, since we're not in react-land here.

File: react/components/enhancedCommerce.ts (hunk 1 above)
Comment (brunojdo) [reply react/components/enhancedCommerce.ts]: I Liked

**Pull request merged**